src/Nest/QueryDsl/MultiTermQueryRewrite/RewriteMultiTerm.cs

```diff
@@ -1,9 +1,13 @@
-﻿using System.Runtime.Serialization;
+﻿using System;
+using System.Runtime.Serialization;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Converters;
 
 namespace Nest
 {
+	/// <summary>
+	/// Multi term query rewrite method
+	/// </summary>
```

Comment:
I'm inclined to remove the enum altogether in `master` as the type can represent the different methods. Also investigate making it a `struct` and implementing equality against `string`